meson.build

```diff
@@ -40,24 +40,7 @@ endif
 
 if get_option('sd-bus-provider') == 'auto'
 	assert(get_option('auto_features').auto(), 'sd-bus-provider must not be set to auto since auto_features != auto')
-	sdbus = dependency('libsystemd',
-		required: false,
-		not_found_message: 'libsystemd not found, trying libelogind',
-	)
-	if not sdbus.found()
-		sdbus = dependency('libelogind',
-			required: false,
-			not_found_message: 'libelogind not found, trying basu',
-		)
-	endif
-	if not sdbus.found()
-		sdbus = dependency('basu',
-			required: false,
-		)
-	endif
-	if not sdbus.found()
-		error('Neither libsystemd, nor libelogind, nor basu was found')
-	endif
+	sdbus = dependency(['libsystemd', 'libelogind', 'basu'])
```

Comment:
The meson docs seem to imply this should be multiple arguments rather than a list, but they don't actually provide an example. Does the list have a different behavior, or do they both just work?

Reply:
Hm, good question. They appear to work the same. I switched to the multiple arg approach since that's what the docs use.

(Note, Sway and seatd use a list right now.)

Reply:
I found the code in question and it seems like it magically collects variadic args into a list, so a list also happens to work: https://github.com/mesonbuild/meson/commit/4deeb907b613760754c5be20e1af019d76b78515#diff-abffc4d23a79ba9faffdc902d5347cccf4d0eef69ee7503b7ad3c6f86a6fc78eR1496

Not sure if this is actually intentional.